packages/server/src/routes/agencies.js

```diff
@@ -26,6 +27,39 @@ router.put('/:agency', requireAdminUser, async (req, res) => {
     res.json(result);
 });
 
+router.delete('/del/:agency', requireAdminUser, async (req, res) => {
+    const { agency } = req.params;
+
+    const { parent, name, abbreviation, warningThreshold, dangerThreshold, } = req.body;
+    const result = await deleteAgency(agency, parent, name, abbreviation, warningThreshold, dangerThreshold,);
+    res.json(result);
+});
+
+router.put('/name/:agency', requireAdminUser, async (req, res) => {
+    const { agency } = req.params;
+
+    const { name } = req.body;
+    const result = await setAgencyName(agency, name);
+    res.json(result);
+});
+
+router.put('/abbr/:agency', requireAdminUser, async (req, res) => {
+    const { agency } = req.params;
+
+    const { abbreviation } = req.body;
+    const result = await setAgencyAbbr(agency, abbreviation);
+    res.json(result);
+});
+
+router.put('/parent/:agency', requireAdminUser, async (req, res) => {
+    const { agency } = req.params;
+
+    // const { parentId } = req.body.parentId;
+    // console.log('reeeeeeqqqqq  ' + req.body.parentId);
+    const result = await setAgencyParent(agency, req.body.parentId);
```

Comment:
Might want to cast `req.body.parentId` as a Number see [here](https://github.com/usdigitalresponse/usdr-gost/pull/116/files#diff-067094b50695ccd7d6188f26c341f3d1cd1213ffc6a762abcb069b5a7db841c4L38)